docs/source/releases.rst

```diff
@@ -1,6 +1,36 @@
 Release History
 ###############
 
+v1.3.0 (2020-07-01)
+===================
+
+Features
+--------
+- Pack camviewer config into a camviewer namespace for ease of access
+  and to avoid name collisions with other data sources.
+
+
+v1.2.3 (2020-05-29)
+===================
+
+Fixes and Maintenance
+---------------------
+- Fix issue with tests freezing
+
```

Comment:
Spacing's kinda inconsistent throughout the file but I think the double space after each release looks better.
```suggestion


```